cldf/requirements.txt

```diff
@@ -5,45 +5,51 @@ chardet==4.0.0
 cldfbench==1.7.1
 cldfcatalog==1.3.2
 clldutils==3.9.0
-colorlog==4.8.0
-csvw==1.10.1
+colorlog==5.0.1
+csvw==1.11.0
+cycler==0.10.0
 decorator==4.4.2
 gitdb==4.0.7
 greenlet==1.1.0
 idna==2.10
 iniconfig==1.1.1
 isodate==0.6.0
-lingpy==2.6.8
+kiwisolver==1.3.1
+lingpy==2.6.7
 Markdown==3.3.4
+matplotlib==3.4.2
```

Comment:
Note that we now also capture the environment necessary to run the analysis code.